### PR TITLE
[Merged by Bors] - feat: port Archive.OxfordInvariants.2021summer.Week3P1

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
I had to change the name from "2021summer" to "Summer2021" because apparently lake doesn't handle file names starting with numbers well..

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Comment by jcommelin** [comment]
bors d+

File: Archive/OxfordInvariants/Summer2021/Week3P1.lean [hunk 1]
```diff
@@ -72,17 +72,17 @@ natural.
   * Divisibility condition:
     `aₙ₊₁ * ((aₙ + aₙ₊₂)/aₙ₊₁ * b - (aₙ * b - a₀)/aₙ₊₁) - a₀ = aₙ₊₁aₙ₊₂b` is divisible by `aₙ₊₂`.
 -/
-
+example (n : ℕ)  : n + 1 ≤ Nat.succ (Nat.succ n)  := by exact Nat.le_succ (n + 1)
```
Comment (jcommelin) [inline Archive/OxfordInvariants/Summer2021/Week3P1.lean:75]:
```suggestion
```

**Comment by jcommelin** [comment]
Sorry for the bors mess, I clicked the wrong buttons in the wrong place.

Should be good to go now.

bors r+

**Pull request closed without merging**